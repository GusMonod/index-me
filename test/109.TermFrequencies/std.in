I like to test
I like to like to test
I like to like to test test
I like to test test test
I like to test test test test test
I like to test test test test test test test test test test
