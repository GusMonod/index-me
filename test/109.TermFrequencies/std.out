I: idf=0.00, in docs: 0(tf = 1), 1(tf = 1), 2(tf = 1), 3(tf = 1), 4(tf = 1), 5(tf = 1)
like: idf=0.00, in docs: 0(tf = 1), 1(tf = 2), 2(tf = 2), 3(tf = 1), 4(tf = 1), 5(tf = 1)
to: idf=0.00, in docs: 0(tf = 1), 1(tf = 2), 2(tf = 2), 3(tf = 1), 4(tf = 1), 5(tf = 1)
test: idf=0.00, in docs: 0(tf = 1), 1(tf = 1), 2(tf = 2), 3(tf = 3), 4(tf = 5), 5(tf = 10)
